From 8411ddaa5891af355745c0541892e0649fd8ea8a Mon Sep 17 00:00:00 2001
From: lhelontra <lhe.lontra@gmail.com>
Date: Tue, 25 Jun 2019 14:33:16 -0300
Subject: [PATCH] fixed neon issues

---
 tensorflow/lite/kernels/internal/BUILD | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/tensorflow/lite/kernels/internal/BUILD b/tensorflow/lite/kernels/internal/BUILD
index 5bafcdc00c..7fb7b4a4ef 100644
--- a/tensorflow/lite/kernels/internal/BUILD
+++ b/tensorflow/lite/kernels/internal/BUILD
@@ -23,6 +23,9 @@ NEON_FLAGS_IF_APPLICABLE = select({
     ":arm": [
         "-O3",
         "-mfpu=neon",
+    ],
+     ":armeabi": [
+        "-O3",
     ],
     ":armeabi-v7a": [
         "-O3",
@@ -76,6 +79,13 @@ config_setting(
     },
 )
 
+config_setting(
+    name = "armeabi",
+    values = {
+        "cpu": "armeabi",
+    },
+)
+
 config_setting(
     name = "arm64-v8a",
     values = {
@@ -572,6 +582,9 @@ cc_library(
         ],
         ":arm64-v8a": [
             ":neon_tensor_utils",
+        ],
+         ":armeabi": [
+            ":neon_tensor_utils",
         ],
         ":armeabi-v7a": [
             ":neon_tensor_utils",
-- 
2.20.1

